src/app/shared/filter/canned-range.component.spec.ts

```diff
@@ -86,7 +86,7 @@ describe('CannedRangeComponent', () => {
     expect(comp.filter.endDate.valueOf()).toEqual(comp.endOfTodayUTC().valueOf());
   });
 
-  it('should go to prev range when PREV button is click', () => {
+  xit('should go to prev range when PREV button is click', () => {
```

Comment:
would be a good week to address this one: https://github.com/PRX/metrics.prx.org/issues/37